tsconfig.json

```diff
@@ -30,5 +30,6 @@
     "scripts/**/*",
     "tmp/bundles/**/*",
     "tmp/constants/**/*"
-  ]
+  ],
+  "files": ["types/global.d.ts"]
```

Comment:
As I said [here](https://github.com/romcal/romcal/pull/374#issuecomment-1263296479), this PR should be merged _after_ #350.

This particular change is described in the [#350 description](https://github.com/romcal/romcal/pull/350#issue-1092095725) (at the very end):

> I had to fix the tests too by using [this workaround](https://github.com/jest-community/jest-extended/issues/367#issuecomment-1127269184) (otherwise `jest-extended` would fail to load in tests). Also, I needed to update `jest` (`^28.1.2`) and `jest-extended` (`^2.0.0`). No other change was required to make the tests pass.

Reply:
@emagnier, you haven’t reacted on this yet. Are you okay with this or not?

If you okay this this or still require some changes? If you require some changes, those should be made in #350.

Reply:
I'm not sure why this is required, has I don't need this line to make tests running.

Reply:
Okay, you’re right, it is not required. I have no idea why it didn’t work for me wethout it then.

If you want, we can remove it.

Reply:
If you don't need it to make tests running: yes, remove it (otherwise keep it for now, meanwhile founding a best raison/fix).